src/main/java/Bundle.properties

```diff
@@ -160,8 +162,8 @@ notification.createDataverse={0} was created in {1} . To learn more about what y
 notification.dataverse.management.title=Dataverse Management - Dataverse User Guide
 notification.createDataset={0} was created in {1}. To learn more about what you can do with a dataset, check out the {2}.
 notification.dataset.management.title=Dataset Management - Dataset User Guide
-notification.wasSubmittedForReview={0}, was submitted for review to be published in {1}. Don't forget to publish it or send it back to the contributor\!
-notification.wasReturnedByReviewer={0}, was returned by the curator of {1}. 
+notification.wasSubmittedForReview={0} was submitted for review to be published in {1}. Don''t forget to publish it or send it back to the contributor\!
```

Comment:
@oscardssmith I know the `Don''t` looks weird but it was to fix the "Dont" (no apostrophe) reported in #2128.